tests/integration/llm/download_llm.py

```diff
@@ -27,10 +21,17 @@ def llm_download(model_id, token, allow_patterns):
                         required=False,
                         type=str,
                         help='The HuggingFace token have access to the model')
-    parser.add_argument('--allow-patterns',
+    parser.add_argument(
+        '--allow-patterns',
+        required=False,
+        type=str,
+        default='*.json,*.pt,*.safetensors,*.txt,*.model,*.tiktoken',
```

Comment:
Should it be a list? i.e, Should we need nargs='+' or passing it as string is fine?

Reply:
it expects a string now, see a few lines below where it does the replace/split to turn the string into a list